test/integrate/dubbo/go-server/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang
```

Comment:
Lisence?

Reply:
Okay, I have modified it in the latest commit.